test/WebJobs.Script.Tests/EndToEndTestsBase.cs

```diff
@@ -294,6 +296,62 @@ public EndToEndTestsBase(TTestFixture fixture)
             Assert.Equal(testData, result);
         }
 
+        // Calls functions that bind to supported argument types and update an entity in a table.
+        protected async Task ApiHubTableTest()
+        {
+            const string TextArg = "text";
+            const string Chars = "ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789";
+
+            // Create the test entity if it doesn't exist.
+            var connection = ConnectionFactory.Default.CreateConnection(ApiHubTestHelper.Key);
+            var tableClient = connection.CreateTableClient();
+            var dataSet = tableClient.GetDataSetReference();
+            var table = dataSet.GetTableReference<ApiHubTestHelper.SampleEntity>(ApiHubTestHelper.TableName);
+            var entity = await table.GetEntityAsync(ApiHubTestHelper.EntityId.ToString());
+            if (entity == null)
+            {
+                await table.CreateEntityAsync(
+                    new ApiHubTestHelper.SampleEntity
+                    {
+                        Id = ApiHubTestHelper.EntityId,
+                        Text = "x"
+                    });
+            }
+
+            // Define function to generate a random text value.
+            var random = new Random();
+            Func<string> newRandomText = () => new string(
+                Enumerable.Repeat('x', 10) // nvarchar(10)
+                    .Select(c => Chars[random.Next(Chars.Length)])
+                    .ToArray());
+
+            // Define a function to verify that the Text column was updated.
+            Func<string, Task> getEntityAndValidateText = async text =>
+            {
+                entity = await table.GetEntityAsync(ApiHubTestHelper.EntityId.ToString());
+                Assert.NotNull(entity);
+                Assert.Equal(text, entity.Text);
+            };
+
+            var arguments = new Dictionary<string, object>();
+            string textArgValue;
+
+            // Test table client binding.
+            arguments[TextArg] = textArgValue = newRandomText();
```

Comment:
True. However, I put them together to avoid concurrency issues between tests given that I am bound by constraints imposed by the ApiHub connectors (cannot create/delete tables). I could replicate that code in 3 separate methods, one updating entity 1, second entity 2 and third entity 3 to avoid concurrency issues but I don't think that will make a huge difference.